mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -1072,3 +1117,40 @@ func flattenTransferSpec(transferSpec *storagetransfer.TransferSpec, d *schema.R
 func usingPosix(transferSpec *storagetransfer.TransferSpec) bool {
 	return transferSpec.PosixDataSource != nil || transferSpec.PosixDataSink != nil
 }
+
+func expandTransferJobNotificationConfig(notificationConfigs []interface{}) *storagetransfer.NotificationConfig {
+	if len(notificationConfigs) == 0 || notificationConfigs[0] == nil {
+		return nil
+	}
+
+	notificationConfig := notificationConfigs[0].(map[string]interface{})
+	var apiData = &storagetransfer.NotificationConfig{
+		PayloadFormat: notificationConfig["payload_format"].(string),
+		PubsubTopic:   notificationConfig["pubsub_topic"].(string),
+	}
+
+	if notificationConfig["event_types"] != nil {
+		apiData.EventTypes = convertStringArr(notificationConfig["event_types"].(*schema.Set).List())
+		log.Printf("[DEBUG] event_types: %v\n\n", apiData.EventTypes)
```

Comment:
I think we can remove the debug logs in this file now

Reply:
Good cache. I will remove it.